pkgs/applications/video/mkvtoolnix/default.nix

```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
     expat
     file xdg_utils boost libebml zlib
     libmatroska libogg libvorbis flac
-    (optional withGUI qt5.qtbase)
+    (optional withGUI qt5.qtbase.dev)
   ];
```

Comment:
Actually you should just move it outside the list, like:

````
] ++ optional withGUI qt5.qtbase;
````

...because now you have a one-element list containing qt5.qtbase in buildInputs, which confuses mkDerivation.